docs/2_2_common_schema.adoc

```diff
@@ -998,12 +998,12 @@ _For the sake of simplicity in the following latexmath:[\dot{m}_i] denotes an in
 
 latexmath:[0 = \sum{\dot{m}_i}]
 
-_Connecting a single <<output>> `outflow` to a single <<input>> `inflow`, or vice versa automatically fulfills the flow constraint, while connecting two single signals of the same flow type requires a negation of the signal.
+_[Connecting a single <<output>> `outflow` to a single <<input>> `inflow`, or vice versa automatically fulfills the flow constraint, while connecting two single signals of the same flow type requires a negation of the signal._
 
-`inflow` and `outflow` is only used as a sign convention for scalar flow quantities which obey Kirchhoff's current law (sum up to zero).
-Other, none scalar, quantities which also sum up to zero, like a mechanical force in 3D space according to D'Alembert's principle, are not covered by this sign convention.
-This is the case since Kirchhoff's current law only holds for scalars where a sign convention is sufficient.
-Other definitions are beyond the scope of this terminal specification and need clear definition in other specifications on top of this.]_
+_`inflow` and `outflow` is only used as a sign convention for scalar flow quantities which obey Kirchhoff's current law (sum up to zero)._
+_Other, none scalar, quantities which also sum up to zero, like a mechanical force in 3D space according to D'Alembert's principle, are not covered by this sign convention._
```

Comment:
"none scalar"? non-scalar?

Reply:
I checked the web, it seems that it is nonscalar (see nonprofit), but you can find non scalar, non-scalar and nonscalar. Funny. Fixed, I will merge right after.